src/ssl/ServerBump.cc

```diff
@@ -75,3 +75,18 @@ Ssl::ServerBump::sslErrors() const
     return errs;
 }
 
+void
+Ssl::ServerBump::storeEntryError(StoreEntry *e)
+{
+    if (entry) {
+        assert(sc);
+        storeUnregister(sc, entry, this);
+        entry->unlock("Ssl::ServerBump");
+    }
+    entry = e;
+    entry->lock("Ssl::ServerBump");
```

Comment:
We should mark each lock (and each unlock) location in a unique way to avoid guesswork during triage. Squid code already uses the URL-like `#` symbol to separate the primary code context from its lower-level location detail IIRC.

```suggestion
    entry->lock("Ssl::ServerBump#storeEntryError");
```

Reply:
commit 82d319ed0f